packages/DateInput/test/DateInput.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import moment from "moment";
+import { render as renderReactTestingLibrary, configure, fireEvent, wait } from "@testing-library/react";
+import DateInput from "../src";
+
+configure({ testIdAttribute: "data-pka-anchor" });
+
+function render(props = {}) {
+  const handleChange = props.onChange || (() => {});
+  const rendered = renderReactTestingLibrary(
+    <DateInput {...props} onChange={handleChange} data-pka-anchor="dateinput" />
+  );
+  const { rerender } = rendered;
+
+  return {
+    ...rendered,
+    rerender: updatedProps => {
+      rerender(<DateInput {...updatedProps} onChange={handleChange} data-pka-anchor="dateinput" />);
+    },
+  };
+}
+
+describe("DateInput", () => {
+  it("should display empty in input by default", () => {
+    const { getByTestId } = render();
+
```

Comment:
Seems like too much whitespace in these tests since they are only 2 or 3 lines

Reply:
Oh, this is just my habit to write tests in AAA style (Arrange, Act, Assert). Can we leave at is? Probably, I can share tips about AAA in next ShopTalk.